network/yarn/pom.xml

```diff
@@ -86,6 +88,15 @@
               </excludes>
             </filter>
           </filters>
+          <relocations>
+            <relocation>
+              <pattern>com.fasterxml.jackson</pattern>
+              <shadedPattern>org.spark-project.com.fasterxml.jackson</shadedPattern>
+              <includes>
```

Comment:
super nit: I wonder if this is needed; we have it for Jetty but not for Guava, and both end up being relocated the same way.